src/node_exit_code.h

```diff
@@ -0,0 +1,54 @@
+#ifndef SRC_NODE_EXIT_CODE_H_
+#define SRC_NODE_EXIT_CODE_H_
+
+#if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
+
+namespace node {
+#define EXIT_CODE_LIST(V)                                                      \
+  V(NoFailure, 0)                                                              \
```

Comment:
```suggestion
  V(Success, 0)                                                              \
```
nit: I slightly prefer `kSuccess` over `kNoFailure` but feel free to ignore if you feel the other way